### PR TITLE
Add documentation for consumption of daily dev builds

### PR DESCRIPTION
**Pull request by chidozieononiwu**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -121,6 +121,15 @@ In some cases, you might want to test against the latest versions of the client
 
 If you make a public API change `eng\Export-API.ps1` script has to be run to update public API listings.
 
+## Dev Feed
+We publish nightly built packages to a dev feed which can be consumed by adding the dev feed blob storage as a package source in visual studio. Follow instruction provided [here](https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio#package-sources). Use https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json  as the source.
```
Comment (maririos) [inline CONTRIBUTING.md]:
```suggestion
We publish nightly built packages to an Azure devOps dev feed which can be consumed by adding the dev feed as a package source in Visual Studio. 

Follow instruction provided [here](https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio#package-sources) and  use `https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json  as the source`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -121,6 +121,15 @@ In some cases, you might want to test against the latest versions of the client
...
+
+You can also achieve this from the command line.
+
+`nuget.exe sources add -Name “Azure SDK for Net Dev Feed” -Source “https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json”`
```
Comment (maririos) [inline CONTRIBUTING.md]: It might be better to put the line with ``` instead of a single ` (format)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -121,6 +121,15 @@ In some cases, you might want to test against the latest versions of the client
...
+
+You can then consume packages from this package source.
```
Comment (maririos) [inline CONTRIBUTING.md]: should we add something about remember to check the `Include prerelease ` box in VS when searching for the packages? => https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages#installing-and-updating-pre-release-packages

**Review by maririos — APPROVED** [review]
LGTM, some minor modifications

File: CONTRIBUTING.md (hunk 1 above)
Comment (chidozieononiwu) [reply CONTRIBUTING.md]: Actually we are not using an Azure DevOps feed for dotnet. We are using a blob storage.

Comment (maririos) [reply CONTRIBUTING.md]: lol! things have changed! good to know. Then NV :)

**Pull request merged**